aait/backend/group-2A/Domain/Entities/Comment.cs

```diff
@@ -8,8 +8,8 @@ public class Comment
         //The User That Commented
         public required int UserId { get; set; }
```

Comment:
you can remove required keyword if you have validation for creating and updating